packages/code-studio/src/main/UserLayoutUtils.ts

```diff
@@ -137,16 +135,18 @@ export async function getDefaultLayout(
     : DEFAULT_LAYOUT_CONFIG_NO_CONSOLE;
 }
 
-export function exportLayout(data: {
-  filterSets: FilterSet[];
-  links: Link[];
-  layoutConfig: ItemConfigType[];
-}): ExportedLayoutV2 {
-  const { filterSets, layoutConfig, links } = data;
+export function exportLayout(
+  data: CustomizableWorkspaceData
+): ExportedLayoutV2 {
+  const { filterSets, layoutConfig, links, pluginDataMap } = data as Omit<
+    ExportedLayoutV2,
+    'version'
+  >;
```

Comment:
I guess this cast is because the actual data types are part of the dashboard package, but the redux package just marks them as `unknown`. We should probably clean that up at some point. Related https://github.com/deephaven/web-client-ui/issues/730